recipes/matrix_client/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "matrix_client" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dce3ccb8665df0d519f08e07a16e6d3f9fab3a947df4b7a7c4bb26573d68f2d5
+
+build:
+  number: 0
+  # pytest-runner 5.1 doesn't exist for py38.
+  skip: true  # [py38]
```

This can probably be noarch?
```suggestion
  noarch: python
```
(It's possible to add the `python != 3.8` constraint if really needed without using `skip` but I'll explain that after you answer my other comment)

---

I had it as `noarch` but the linter comment above was why I removed it.
> * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.